analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/OperationProcessors/References.cs

```diff
@@ -71,10 +71,20 @@ internal sealed class PropertyReference : SimpleProcessor<IPropertyReferenceOper
     protected override IPropertyReferenceOperationWrapper Convert(IOperation operation) =>
         IPropertyReferenceOperationWrapper.FromOperation(operation);
 
-    protected override ProgramState Process(SymbolicContext context, IPropertyReferenceOperationWrapper propertyReference) =>
-        propertyReference.Instance.TrackedSymbol() is { } symbol
-            ? context.SetSymbolConstraint(symbol, ObjectConstraint.NotNull)
-            : context.State;
+    protected override ProgramState Process(SymbolicContext context, IPropertyReferenceOperationWrapper propertyReference)
```

Comment:
Educational: explicit cast, while calling `.Value`, is not supported by this because the cast is a method call at CFG level (is it?) and SE is not cross-level. So you will need to learn the same you learn here (i.e. `Value` as a non-null value) in a "IInvocationOperationWrapper". Correct?

Reply:
All casts are represented as `ConversionOperation` (implicit or explicit). That will be handled later.

`InvocationOperation` is not relevant here.